src/Http/Http.Results/test/ResultsTests.cs

```diff
@@ -980,15 +980,15 @@ public void Problem_WithArgs_ResultHasCorrectValues()
     [Theory]
     [InlineData(StatusCodes.Status400BadRequest, "Bad Request", "https://tools.ietf.org/html/rfc9110#section-15.5.1")]
     [InlineData(StatusCodes.Status418ImATeapot, "I'm a teapot", null)]
-    [InlineData(499, null, null)]
+    [InlineData(499, "Client Closed Request", null)]
```

Comment:
@Tratcher  It will be 498 or 499 ?
b) If I will change it to null then the test got failed because now w e have description for 499 status code.

![image](https://user-images.githubusercontent.com/83835739/219045929-96f17484-9032-4afd-9395-bd3afc021799.png)

Reply:
This test used to test an unkown status code (499). Since you're now making 499 a known status code then we need to pick a different unkown status code. I'd suggested 498 since we don't define that one.